ios-template/App/Podfile

```diff
@@ -11,6 +11,19 @@ def assertDeploymentTarget(installer)
   end
 end
 
+# workaround for Xcode 15 and Cocoapods 1.21.1
+# Should not be needed after Cocoapods 1.13.0
+def fixToolChainDirectory(installer)
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+    xcconfig_path = config.base_configuration_reference.real_path
+    xcconfig = File.read(xcconfig_path)
+    xcconfig_mod = xcconfig.gsub(/DT_TOOLCHAIN_DIR/, "TOOLCHAIN_DIR")
+    File.open(xcconfig_path, "w") { |file| file << xcconfig_mod }
```

Comment:
Not to be _that_ guy, but this block should probably be indented for style's sake